templates/zerver/integrations/github-actions.md

```diff
@@ -1,8 +1,27 @@
-It's easy to send Zulip messages from GitHub Actions! This is useful:
+# Zulip GitHub Actions integration
 
-  * If you want to report the success or failure of an Action to Zulip.
-  * If you want to send output from your actions to Zulip.
+It's easy to send Zulip messages from GitHub Actions!
 
-Instructions are available [here](https://github.com/zulip/github-actions-zulip#readme).
+{start_tabs}
 
-See also the [GitHub integration](/integrations/doc/github).
+1. {!create-a-generic-bot.md!}
```

Comment:
As recommended by Tim Abbott in [CZO](https://chat.zulip.org/#narrow/channel/19-documentation/topic/integration-docs.3A.20Using.20tabbed.20sections.20for.20a.20single.20instruc/near/1997524),
- Added 2 new instructions.
- Rewrote the reference instruction (the last one), made the link more specific.
- Added links to the Related Docs section.